test/test_nestedtensor.py

```diff
@@ -2003,6 +2003,25 @@ def test_reshape(self, device, dtype):
             lambda: nt1.reshape(2, -1, -1, 2, 2)
         )
 
+    @dtypes(torch.float, torch.float16, torch.double)
+    def test_narrow(self, device, dtype):
```

Comment:
Have you tried narrowing some non-contiguous NestedTensors? It's also ok to disable non-contiguous NestedTensors for now for this narrow.

Reply:
Added error test case for non-contiguous input